README.md

```diff
@@ -45,9 +45,18 @@ are being created:
 # Cleanup
 
 In cases where the beaker process is killed before finishing, it may leave resources in Hetzner cloud. These will need to be manually deleted.
-
 Look for servers in your project named exactly as the ones in your beaker host configuration and SSH keys with names beginning with `Beaker-`.
 
+The gem will try to automatically delete old virtual machines. Every created
+cloud instance gets a label `delete_vm_after: 'Fri, 22 Sep 2023 15:39:17 +0200'`.
```

Comment:
Timestamp formats here (human-readable with TZ) and in the unit tests below (unix time w/o TZ) are different. Is it expected?